components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -530,6 +499,87 @@ func (backend *ES2Backend) GetReport(esIndex string, reportId string,
 	return report, errorutils.ProcessNotFound(nil, reportId)
 }
 
+func convertControl(profileControlsMap map[string]*reportingapi.Control, reportControlMin ESInSpecReportControl, filters map[string][]string) *reportingapi.Control {
+	profileControl := profileControlsMap[reportControlMin.ID]
+
+	if profileControl != nil {
```

Comment:
There doesn't appear to be an `else` on this condition, so you could remove a level of nesting by putting a guard in front checking for nil and returning.